src/handlers/int-test-support/helpers/snsHelper.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  PublishCommand,
+  PublishCommandOutput,
+  PublishInput,
+} from "@aws-sdk/client-sns";
+import { resourcePrefix, runViaLambda } from "./envHelper";
+import { snsClient } from "../clients";
+import { sendLambdaCommand } from "./lambdaHelper";
+
+let snsParams: PublishInput;
+
+const snsParameters = async (
+  snsValidEventPayload: any
+): Promise<{
+  Message: string;
+  TopicArn: string;
+}> => ({
+  Message: JSON.stringify(snsValidEventPayload),
+  TopicArn: `arn:aws:sns:eu-west-2:582874090139:${resourcePrefix()}-test-TxMA-topic`,
```

Comment:
I know this is fixed and unlikely to change, but it still seems wrong to have the acct # hardcoded in a public repo.  Should this be read from config?

Reply:
This also means that this test will not work in the build account (where it should be able to run), so I'll have to fix this somehow...

Reply:
Added node-cache and look this ARN in run-time now (once).